python/GafferUI/EditScopeUI.py

```diff
@@ -406,8 +409,33 @@ def __dropNode( self,  event ) :
 		else:
 			return None
 
+	def __buttonPress( self, widget, event ) :
+
+		return event.buttons == event.Buttons.Middle and self.__editScope() is not None
```

Comment:
I think we could accept `Buttons.Left` here is well, to make this a little more discoverable. Because we're handling the event on `self`, this doesn't prevent left-click working on the menu button as usual, because it has first dibs.

Reply:
Added in 4347523. My initial hesitation was the left-drag feeling too much like "I'm linking the Edit Scopes of these two editors", but as this interaction is already fairly esoteric, making it more discoverable is worthwhile...